tests/test_2D.py

```diff
@@ -0,0 +1,21 @@
+import subprocess
+import sys
+from pathlib import Path
+
+from .util import get_data
+
+
+def test_output_2D_file_format() -> None:
+    """A simple test for format tag fix for 2D output arrays."""
+
+    Path("filename.txt").touch()
+    params = [
+        sys.executable,
```

Comment:
It would be better to use `get_main_output()` from https://github.com/common-workflow-language/cwltool/blob/46e0405320014d3d9e74c1047337e81ee2adc89c/tests/util.py#L85C5-L85C16